test/MusicStore.Test/ShoppingCartControllerTest.cs

```diff
@@ -27,6 +30,8 @@ public ShoppingCartControllerTest()
                       .AddInMemoryStore()
                       .AddDbContext<MusicStoreContext>();
 
+            services.AddMvc();
```

Comment:
Nope that would not work. The OOB attribute only understands a form value of ```__RequestVerificationToken```. 
What we should do : 
Use the ```[ValidateAntiForgeryToken]``` and in our ajax request we should post with a key of ```__RequestVerificationToken``` and a value of just the form token. That should work and should be secure.